tests/document.rs

```diff
@@ -1,4 +1,1019 @@
 #![allow(unused_imports)]
 #![allow(unused_parens)]
 
+use log::trace;
+use pretty_assertions::assert_eq;
+use serde_json::{json, Value};
+
+use arangors::{
+    document::{
+        DocumentInsertOptions, DocumentOverwriteMode, DocumentReadOptions, DocumentRemoveOptions,
+        DocumentReplaceOptions, DocumentResponse, DocumentUpdateOptions,
+    },
+    ClientError, Connection, Document,
+};
+use common::{get_arangodb_host, get_normal_password, get_normal_user, test_setup};
+use std::ptr::null;
+
 pub mod common;
+
+#[cfg(not(feature = "arango3_7"))]
+#[maybe_async::test(
+    any(feature = "reqwest_blocking"),
+    async(any(feature = "reqwest_async"), tokio::test),
+    async(any(feature = "surf_async"), async_std::test)
+)]
+async fn test_post_create_document() {
+    test_setup();
+    let host = get_arangodb_host();
+    let user = get_normal_user();
+    let password = get_normal_password();
+
+    let collection_name = "test_collection_create_document";
+
+    let conn = Connection::establish_jwt(&host, &user, &password)
+        .await
+        .unwrap();
+    let mut database = conn.db("test_db").await.unwrap();
+
+    let coll = database.drop_collection(collection_name).await;
+    assert_eq!(coll.is_err(), true);
```

Comment:
This test function are for post_create_document, so don't need to assert for collection or anything unrelated to the aim of this test function. This functionality is tested elsewhere in drop_collection. You can use `unwrap` here.

But for this specific case, I suggect use `expect("should creata a collection for test, but it already exists")`.